Dockerfile.melodic

```diff
@@ -19,13 +19,14 @@ ARG CARTOGRAPHER_VERSION=master
 # We require a GitHub access token to be passed.
 ARG github_token
 
-# Bionic's base image doesn't ship with sudo.
-RUN apt-get update && apt-get install -y sudo
+# Bionic's base image doesn't ship with sudo and curl.
+RUN apt-get update && apt-get install -y sudo curl
 
 # First, we invalidate the entire cache if cartographer-project/cartographer has
 # changed. This file's content changes whenever master changes. See:
 # http://stackoverflow.com/questions/36996046/how-to-prevent-dockerfile-caching-git-clone
-ADD https://api.github.com/repos/cartographer-project/cartographer/git/refs/heads/master?access_token=$github_token \
+RUN mkdir -p cartographer_ros && \
+    curl -v -H "Authorization: ${github_token}" https://api.github.com/repos/cartographer-project/cartographer/git/refs/heads/master > \
```

Comment:
```suggestion
ADD https://api.github.com/repos/cartographer-project/cartographer/git/refs/heads/master \
```

Reply:
thanks for pointing this out. The problem without providing a token is that you are rate limited to 60 requests per hour per IP and since the IPs are shared on the CI, the rate limit will be hit on a a randomly basis and the CI will fail.
I think I have now found a solution / workaround for this issue.

Reply:
LGTM